### PR TITLE
fixes #108 : Inventory added

### LINKED ISSUE #108: Inventory for Player
### Expected results
- Ability to collect items in the game and use them

### Actual results
- Completely isolated player

### Steps to reproduce the issue
- Adding inventory mechanism to game

## Would you like to work on this
- YES 

*after approval*

### PR DESCRIPTION
**Pull request by YashKumarVerma**
This pull request is related to issue #108 

I modified the labyrinth, you can test my changes here:
http://rawgit.com/yashkumarverma/labyrinth/inventory/index.html

The problem I want to solve is **that there is no inventory for user**

My solution for the problem is **i added the basis concept of inventory**

image

### CODE REVIEW

File: js/player.js [hunk 1]
```diff
@@ -10,6 +10,23 @@ function Player() {
...
+  // INVENTORY SYSTEM
+  // add item to players inventory
+  this.addToInventory = function(item) {
+    this.inventory.push(item);
+  };
+  // check for item in players inventory
+  this.hasInInventory  = function(item) {
+    for (var i = this.inventory.length - 1; i >= 0; i--) {
+      if(this.inventory[i] === item){
+        return 1;
```
Comment (niccokunzmann) [inline js/player.js]: should be true

File: js/player.js [hunk 2]
```diff
@@ -10,6 +10,23 @@ function Player() {
...
+      }else{
+        return 0;
```
Comment (niccokunzmann) [inline js/player.js]: should be false

**Comment by niccokunzmann** [comment]
Thanks!

**Comment by YashKumarVerma** [comment]
@niccokunzmann i sent in a PR correcting those 😢

**Comment by YashKumarVerma** [comment]
i think that can be a `good first issue`

**Pull request merged**